tests/components/scene/test_init.py

```diff
@@ -152,3 +146,26 @@ async def test_services_registered(hass):
     assert hass.services.has_service("scene", "reload")
     assert hass.services.has_service("scene", "turn_on")
     assert hass.services.has_service("scene", "apply")
+
+
+async def setup_lights(hass, entities):
+    """Set up the light component."""
+    assert await async_setup_component(
+        hass, light.DOMAIN, {light.DOMAIN: {"platform": "test"}}
+    )
+    await hass.async_block_till_done()
+
+    light_1, light_2 = entities
+
+    await hass.services.async_call(
+        "light",
+        "turn_off",
+        {"entity_id": [light_1.entity_id, light_2.entity_id]},
+        blocking=True,
+    )
+    await hass.async_block_till_done()
+
+    assert not light.is_on(hass, light_1.entity_id)
+    assert not light.is_on(hass, light_2.entity_id)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)